test/blackbox-tests/test-cases/pkg/multiple-opam-repos.t

```diff
@@ -38,8 +38,7 @@ We have to define both repositories in the workspace, but will only use `new`.
   > (context
   >  (default
   >   (name default)
-  >   (solver_env
-  >    (repositories new))))
+  >   (fallback_repositories new)))
```

Comment:
Likewise. What are fallback repositories? What are they falling back from? Where can I specify normal repositories? Even as someone involved in dune development, this confuses me.